src/module/actor/creature/sheet.ts

```diff
@@ -475,6 +475,17 @@ export abstract class CreatureSheetPF2e<TActor extends CreaturePF2e> extends Act
                 await effect.increase();
             }
         });
+
+        // Change whether an effect is secret to players or not
+        $html.find("a[data-action=effect-toggle-secret]").on("click", async (event) => {
```

Comment:
We're kinda moving off of JQuery for these. Can you use vanilla JS and htmlClosest() for this? There should be examples if you follow the references?

Reply:
Replaced with a combination of `htmlQueryAll` and `htmlClosest`.